DeepSpeech.py

```diff
@@ -432,14 +491,13 @@ def train():
             FLAGS.augmentation_speed_up_std > 0 or
             FLAGS.augmentation_sparse_warp):
         do_cache_dataset = False
-
+    
```

Comment:
nit: empty spaces

Reply:
the indentation is level with the block

Reply:
please get rid of all white-spaces changes

Reply:
I'm confused -- does this mean I deleted a newline *and* added a tab, or that I just added a tab to an existing line?

Reply:
nvm -- there's no block indentation inside DS functions on newlines... most interesting indeed

Reply:
fixed in 75e75498d304ef743c23367056c315c2a4ab3d35